### PR TITLE
MC-41075: Updated several topics of the user guides site

### PR DESCRIPTION
**Pull request by danidelcar**
## Purpose of this pull request

This pull request (PR) updates several pages from the user guide site. Correspond to several issues requested by PO office for CatalogInventory. However, we are still missing on information on 2 issues:

JIRA [MC-41074](https://jira.corp.magento.com/browse/MC-41074)
JIRA [MC-41076](https://jira.corp.magento.com/browse/MC-41076)

## Affected DevDocs pages

This PR modifies the following pages:

- [catalog/products](https://docs.magento.com/user-guide/catalog/products.html).
- [catalog/product-create-simple](https://docs.magento.com/user-guide/catalog/product-create-simple.html).
- [catalog/inventory-stock](https://docs.magento.com/user-guide/catalog/inventory-stock.html).
- [catalog/inventory-product-stock-options](https://docs.magento.com/user-guide/catalog/inventory-product-stock-options.html).
- [catalog/inventory-configure-inventory-management](https://docs.magento.com/user-guide/catalog/inventory-configure-inventory-management.html).

## Other information

Track work through JIRA [MC-41075](https://jira.corp.magento.com/browse/MC-41075)

### CODE REVIEW

**Review by danipt — APPROVED** [review]

File: src/catalog/product-create-simple.md [hunk 1]
```diff
@@ -145,4 +145,6 @@ Scroll down and complete the information in the following sections as needed:
 
 - Simple products can be included in configurable, bundle, and grouped product types.
 
+- Simple product configuration overwrites configurable product configurations for a specific product.
```
Comment (keharper) [inline src/catalog/product-create-simple.md:148]:
```suggestion
- Simple product configuration overrides configurable product configurations for a specific product.
```

File: src/catalog/products.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ redirect_from:
 
 All products in the catalog are accessible from the [Products]({% link catalog/products.md %}) page, where you can create new products and edit existing ones. For a multi-site installation, each website can offer a different selection of products for sale from the same catalog.
 
-The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B installations with [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
+The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B installations with the [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
```
Comment (keharper) [inline src/catalog/products.md:9]: Remove the "the" you added. Change Magento Commerce for B2B to a variable.

File: src/catalog/inventory-product-stock-options.md [hunk 3]
```diff
@@ -92,11 +95,14 @@ Configure the quantities and additional settings per [added source]({% link cata
...
 | Allow Multiple Boxes for Shipping | Global | Determines if parts of the product can be shipped separately.  |
 | Backorders | Global | Determines how  backorders are managed. Backorders do not change the processing status of the order. Funds are still authorized or captured immediately when the order is placed, regardless of whether the product is in stock. Products are shipped as they become available. When enabled, we recommend entering a negative amount for the Out-of-Stock Threshold. Options:<br />**No Backorders** -  Does not accept backorders when product is out of stock.<br />**Allow Qty Below 0** - Accepts backorders when the quantity falls below zero.<br />**Allow Qty Below 0 and Notify Customer** - Accepts backorders when the quantity falls below zero, but notifies customers that orders can still be placed. |
 | Enable Qty Increments | Global | Determines if the product can be sold in quantity increments. |
+
+{:.bs-callout .bs-callout-info}
+Simple product configuration overwrites configurable product configurations for a specific product.
```
Comment (keharper) [inline src/catalog/inventory-product-stock-options.md:108]:
```suggestion
Simple product configuration overrides configurable product configurations for a specific product.
```

**Review by keharper — APPROVED** [review]
Approved, pending some minor revisions.

File: src/catalog/products.md (hunk 2 above)
Comment (danidelcar) [reply src/catalog/products.md:9]: You mean adding it like this: `<span class="b2b-only">Magento Commerce for B2B</span>` ?

Comment (keharper) [reply src/catalog/products.md]: I'm not sure how variables work in merchdocs, but in devdocs, it would be `{{ site.data.var.b2b }}`.

Comment (leslietilling) [reply src/catalog/products.md]: We don't use the `site.data` variable for internal UG links.

File: src/catalog/products.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ redirect_from:
 
 All products in the catalog are accessible from the [Products]({% link catalog/products.md %}) page, where you can create new products and edit existing ones. For a multi-site installation, each website can offer a different selection of products for sale from the same catalog.
 
-The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B installations with [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
+The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In <span class="b2b-only">Magento Commerce for B2B</span> installations with [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
```
Comment (leslietilling) [inline src/catalog/products.md:9]: I don't think we need the span class here, as it is stated explicitly. 

```suggestion
The Products grid lists all products in the catalog, indicates the website(s) where they are available, and if they are currently enabled for sale. In Magento Commerce for B2B, installations with [Shared Catalog]({% link catalog/catalog-shared.md %}) enabled, the grid includes a column that indicates which products have alternate discount pricing in a shared catalog.
```

**Review by leslietilling — APPROVED** [review]

**Comment by danidelcar** [comment]
running tests

**Comment by danidelcar** [comment]
@leslietilling PR is ready to be merged, but does not authorize me to merge it

**Comment by leslietilling** [comment]
> @leslietilling PR is ready to be merged, but does not authorize me to merge it

This is the public repo, so it must be imported to our team fork and merged from there.

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Comment by leslietilling** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1379`.

**Pull request merged**